### ISSUE TITLE
svelte template no longer compiles due to vite-plugin-svelte change

### ISSUE THREAD

**Issue by kelvie** [wailsapp/wails#1579]
### Description

When you create a new project using the svelte template, it doesn't compile right away due to vite-plugin-svelte-1.0.1:

```
wails dev                                                                                                                            254 [23:37:58]
Wails CLI v2.0.0-beta.38


Executing: go mod tidy -compat=1.17
Executing: wails generate module
Building application for development...
  - Installing frontend dependencies:     
    npm ERR! code ERESOLVE
    npm ERR! ERESOLVE unable to resolve dependency tree
    npm ERR! 
    npm ERR! While resolving: frontend@0.0.0
    npm ERR! Found: vite@2.9.14
    npm ERR! node_modules/vite
    npm ERR!   dev vite@"^2.9.9" from the root project
    npm ERR! 
    npm ERR! Could not resolve dependency:
    npm ERR! peer vite@"^3.0.0" from @sveltejs/vite-plugin-svelte@1.0.1
    npm ERR! node_modules/@sveltejs/vite-plugin-svelte
    npm ERR!   dev @sveltejs/vite-plugin-svelte@"^1.0.0-next.30" from the root project
    npm ERR! 
    npm ERR! Fix the upstream dependency conflict, or retry
    npm ERR! this command with --force, or --legacy-peer-deps
    npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
    npm ERR! 
    npm ERR! See /home/kelvie/.npm/eresolve-report.txt for a full report.
    
    npm ERR! A complete log of this run can be found in:
    npm ERR!     /home/kelvie/.npm/_logs/2022-07-17T06_38_08_374Z-debug-0.log
    

Build error - continuing to run current version
exit status 1
Running frontend dev watcher command: 'npm run dev'
Watching (sub)/directory: /home/kelvie/Source/wailstest
Using DevServer URL: http://localhost:34115
Using Frontend DevServer URL: http://localhost:3000
Using reload debounce setting of 100 milliseconds

> frontend@0.0.0 dev
> vite

/tmp/dev-a5935b77.sh: line 1: vite: command not found
Error from 'npm run dev': exit status 127
Dev command exited!
wails ^C
Caught quit
Dev command killed!


ERROR: remove : no such file or directory

If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony
```

### To Reproduce

1. `wails init -n wailstest -t svelte`
2. `cd wailstest`
3. `wails dev`

### Expected behaviour

To not return an error.

### System Details

```shell
Wails CLI v2.0.0-beta.38


Scanning system - Please wait (this may take a long time)...Done.

System
------
OS:             Arch Linux
Version:        Unknown
ID:             arch
Go Version:     go1.18.3
Platform:       linux
Architecture:   amd64

Wails
------
Version:                v2.0.0-beta.38
Package Manager:        pacman

Dependency      Package Name    Status          Version
----------      ------------    ------          -------
*docker         docker          Installed       1:20.10.17-1
gcc             gcc             Installed       12.1.0-2
libgtk-3        gtk3            Installed       1:3.24.34-1
libwebkit       webkit2gtk      Installed       2.36.4-2
npm             npm             Installed       8.14.0-1
pkg-config      pkgconf         Installed       1.8.0-1

* - Optional Dependency

Diagnosis
---------
Your system is ready for Wails development!


If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony
```
```


### Additional context

It looks like as of v1.0.1 (released 4 days ago), vite-plugin-svelte uses vite 3.0.0 https://github.com/sveltejs/vite-plugin-svelte/commit/925c3b2fbd422e1b5d528a4c7cfb4fe8a8efae66, which I guess this doesn't support yet.

**Comment by leaanthony** [comment]
Thanks for reporting this. I guess we should be using `npm ci` instead of `npm install` by default....

**Comment by leaanthony** [comment]
I've got a draft PR here: https://github.com/wailsapp/wails/pull/1580. There appears to be an error thrown by Go but it does not appear to affect functionality. Investigating 👍

**Comment by ianmjones** [comment]
@kelvie this is fixed for me in v2.0.0-beta.42, is it ok for you too?

**Comment by kelvie** [comment]
@ianmjones Looks like it is fixed. Thank you!.
